sound/soc/sof/intel/hda-dai.c

```diff
@@ -400,6 +400,49 @@ static const struct snd_soc_dai_ops hda_link_dai_ops = {
 	.prepare = hda_link_pcm_prepare,
```

Comment:
The lower case "dai" pops up a bit in commit message. If you look at git log history for ASoC, it's all mixed and matches (and "hw" is an acronym as well), so maybe this is ok. People will understand what this means.

Reply:
ok changing